File: EqBench/CLEVER/oneN2/EqWMain/oldV.c
int lib(int x){
	if (x > 10)
		return 11;
	else
		return x;
}
int client(int x){
	if (x > lib(x))
		return x;
	else
		return lib(x);
}
int main()
{
return 0;
}

File: EqBench/CLEVER/getSign2/Eq/oldV.java
package benchmarks.CLEVER.getSign2.Eq;
public class oldV {
	private int lib(int x) {
		if (x == 0)
		   return 0;
		if (x < 0)
		   return -1;
		else
		   return 1;
	  }
	public int client(int x){
		if (x > 0) {
		  return lib(x);
		}
		return x;
	  }
}